lib/rspec/rails/fixture_support.rb

```diff
@@ -10,6 +10,10 @@ module FixtureSupport
         include ActiveRecord::TestFixtures
 
         included do
+          setup if RSpec.configuration.use_active_record?
```

Comment:
This was calling another `setup`, specificly the one from `lib/rspec/rails/adapters.rb`.
The following:
```
included do
  next unless RSpec.configuration.use_active_record?

  self.fixture_path = RSpec.configuration.fixture_path
  # the rest of setup
```

works just fine.